internal/command/arguments/extended.go

```diff
@@ -185,7 +231,20 @@ func (o *Operation) Parse() tfdiags.Diagnostics {
 			))
 		}
 	default:
-		o.PlanMode = plans.NormalMode
+		if len(o.ActionTargets) > 0 {
+			o.PlanMode = plans.RefreshOnlyMode
+		} else {
+			o.PlanMode = plans.NormalMode
+		}
+	}
+
+	if !o.Refresh && len(o.ActionTargets) > 0 {
+		// TODO: Check this? Can we trigger actions without refreshing?
```

Comment:
I think we said we wanted to have the option to skip refresh like we do in other commands

Reply:
I will add some tests for this, and enable in a follow up PR.